assets/html/index.html

```diff
@@ -1 +1,22 @@
-
+<html>
+  <head>
+    <title>Challenge One</title>
+  </head>
+  <body>
+    <h1 align="center">This is Chris Featley's page</h1>
+    <p align="center">This should be a photo of Chris</p>
+    <!--I realise I need to actually add a photo here and then do img src -->
```

Comment:
yeh so can you do it then please?

Reply:
added pic that is stored in assets folder